pydoctor/extensions/zopeinterface.py

```diff
@@ -297,18 +313,18 @@ def visit_Call_zope_interface_classImplements(self, funcName: str, node: ast.Cal
                                 funcName == 'zope.interface.classImplementsOnly')
     visit_Call_zope_interface_classImplementsOnly = visit_Call_zope_interface_classImplements
 
-    def visit_ClassDef(self, node: ast.ClassDef) -> Optional[ZopeInterfaceClass]:
-        cls = super().visit_ClassDef(node)
-        if cls is None:
-            return None
-        assert isinstance(cls, ZopeInterfaceClass)
+    def visit_ClassDef(self, node: ast.ClassDef) -> None:
```

Comment:
To keep the same visitor flow as before, we should use `depart_ClassDef` here. But both seem to work.

Reply:
Done.